test_unstructured_ingest/test-ingest-src.sh

```diff
@@ -1,50 +1,56 @@
 #!/usr/bin/env bash
 
-set -eu -o pipefail
+set -u -o pipefail
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+SKIPPED_FILES_LOG=$SCRIPT_DIR/skipped-files.txt
```

Comment:
I assume test-ingest-src and test-ingest-dest do not run parallel in CI, or the skipped-files.txt might be overwritten or deleted while the other is running? Or does the CI actually make completely separate environments for the different tests?

Reply:
They are run in parallel but on different machines so this shouldn't cause any conflict.

Reply:
And that file isn't persisted between tests so it should only live long enough for the script to print it in the logs.